centos/centos-8.x/centos-8.3-hpc/install_intel_libs.sh

```diff
@@ -1,4 +1,9 @@
 #!/bin/bash
+set -ex
 
-$COMMON_DIR/install_intel_libs.sh
-
+# Intel® oneAPI Math Kernel Library
+VERSION="2022.1.0.223"
+ONE_MKL_DOWNLOAD_URL=https://registrationcenter-download.intel.com/akdlm/irc_nas/18721/l_onemkl_p_${VERSION}_offline.sh
+$COMMON_DIR/write_component_version.sh "INTEL_ONE_MKL" $VERSION
```

Comment:
I prefer we write component version after the package is successfully installed, like how we do for AOCL.

Reply:
Added